src/main/java/co/rsk/federate/watcher/FederationWatcherListenerImpl.java

```diff
@@ -51,13 +55,17 @@ public void onProposedFederationChange(Federation newProposedFederation) {
             // start {@code BtcReleaseClient} with proposed federation
             // so it can sign svp spend tx
             btcReleaseClient.start(newProposedFederation);
+
+            // add proposed federation to active btc to rsk client so
+            // it can register svp spend tx in the bridge
+            bitcoinWrapper.addFederationListener(newProposedFederation, btcToRskClientActive);
```

Comment:
This is the main and only change of the PR, the rest are small refactors

Reply:
Isn't this done already in `btcReleaseClient.start`?

Reply:
Nope